### PR TITLE
Random Changes

### PR DESCRIPTION
**Pull request by captaincobbs**
- Organized ConstructibleElement_Industry Icon Files
- Removed a lot of copyrighted images
- Removed unused assets from the first draft of the mod
- Removed Quest template (was causing bugs)
- Fixed Drudge Construction/Upgrade constructibles scoping to ClassEmpire instead of ClassColonizedStarSystem
- Updated many ConstructibleElement_Industry to match changes that have happened in ESG
- Fixed the Mavros getting duplicates of some ESG constructibles / empire improvements

### CODE REVIEW

**Review by FluffiestPrincess — DISMISSED** [review]

File: Localization/english/ES2_Localization_Assets_Locales.xml [hunk 1]
```diff
@@ -76,11 +76,11 @@ Can build and upgrade Drudges using #FF8C00#[custom07]#Revert#, a robotic popula
   <LocalizationPair Name="%PopulationMajorMavrosTitle">Mavros</LocalizationPair>
   <LocalizationPair Name="%PopulationMajorMavrosDescription">TO BE WRITTEN</LocalizationPair>
   <LocalizationPair Name="%PopulationCollectionBonusTraitCustomFactionMavrosTitle">Covetous Collection</LocalizationPair>
-  <LocalizationPair Name="%PopulationCollectionBonusTraitCustomFactionMavrosDescription">TO BE WRITTEN</LocalizationPair>
+  <LocalizationPair Name="%PopulationCollectionBonusTraitCustomFactionMavrosDescription">This collection </LocalizationPair>
```
Comment (FluffiestPrincess) [inline Localization/english/ES2_Localization_Assets_Locales.xml:79]: This seems unfinished. Copy-paste error, or did you just push too early?

**Review by FluffiestPrincess — COMMENTED** [review]
See comments on files. I don't think this merge would break anything that's not already broken, but unless I've missed something, it doesn't fix all the things you thought it did.

File: GUI/GUIElements[StarSystemImprovement].xml [hunk 2]
```diff
@@ -45,8 +45,8 @@
     <Title>%StarSystemImprovementDefense2MavrosTitle</Title>
     <Description>%StarSystemImprovementDefense2MavrosDescription</Description>
     <Icons>
-      <Icon Size="Small" Path="Textures/GUI/PlanetaryCannonSmall.png" />
-      <Icon Size="Large" Path="Textures/GUI/PlanetaryCannonLarge.png" />
+      <Icon Size="Small" Path="Textures/Constructibles/StarSystemImprovementDefense2MavrosSmall.png" />
```
[the remark this answers is not in the document]
Comment (captaincobbs) [reply GUI/GUIElements[StarSystemImprovement].xml:48]: I preferred the image too :(

File: Localization/english/ES2_Localization_Assets_Locales.xml (hunk 1 above)
[the remark this answers is not in the document]
Comment (captaincobbs) [reply Localization/english/ES2_Localization_Assets_Locales.xml:79]: Ah didn't mean to push that, just fixed thanks

File: Simulation/ConstructibleElement_Industry[Scrap].xml [hunk 3]
```diff
@@ -101,7 +101,7 @@
     <SupervisorGain Name="Population" />
     <Tags>NoBuyout</Tags>
     <QueuedSimulationDescriptorReference Name="QueuedDescriptorIncreaseVirtualPopulation"/>
-    <CustomCost ResourceName="EmpireScrap" Instant="true">3 + (Property(StockLocation, @'ClassColonizedStarSystem', DrudgeCount) * 2)</CustomCost>
+    <CustomCost ResourceName="EmpireScrap" Instant="true">3 + (Property(StockLocation, @'ClassColonizedStarSystem', DrudgeWorkerCount) * 2)</CustomCost>
```
[the remark this answers is not in the document]
Comment (captaincobbs) [reply Simulation/ConstructibleElement_Industry[Scrap].xml:104]: Good catch thanks

File: Simulation/ConstructibleElement_Industry[Scrap].xml [hunk 4]
```diff
@@ -101,7 +101,7 @@
     <SupervisorGain Name="Population" />
     <Tags>NoBuyout</Tags>
     <QueuedSimulationDescriptorReference Name="QueuedDescriptorIncreaseVirtualPopulation"/>
-    <CustomCost ResourceName="EmpireScrap" Instant="true">3 + (Property(Context, @'ClassColonizedStarSystem', DrudgeCount) * 2)</CustomCost>
```
[the remark this answers is not in the document]
Comment (captaincobbs) [reply Simulation/ConstructibleElement_Industry[Scrap].xml:104]: image
Works fine for me, but the tooltip hasn't been updated yet

File: Simulation/ConstructibleElement_Science[Mavros].xml [hunk 5]
```diff
@@ -423,43 +422,18 @@
...
-    <PathPrerequisite Flags="Prerequisite,DiplomaticTradable,Discard,Faction,AffinityGameplayTechnologyMajorMavros" Inverted="false">ClassEmpire,AffinityGameplayMajorMavros</PathPrerequisite>
-    <SimulationDescriptorReference Name="TechnologyPopulationSlotsMavros" />
-    <SimulationDescriptorReference Name="TechnologyCountScienceAndExplorationIncreaseStage5" />
-    <SimulationDescriptorReference Name="TechnologyCounter" />
-    <PopulationEvent Name="PopulationPoliticalEventEcologistTechnology" />
-
-    <HelperVariable Name="TurnVariable" />
-    <HelperFormula>40 * $(TurnVariable)</HelperFormula>
-  </TechnologyDefinition>
+  <!-- Arcological Administration -->
```
[the remark this answers is not in the document]
Comment (captaincobbs) [reply Simulation/ConstructibleElement_Science[Mavros].xml:425]: Really dumb mistake, thanks for catching

**Comment by captaincobbs** [comment]
Should be good now

**Review by FluffiestPrincess — APPROVED** [review]

**Pull request merged**